docs-src/concepts/what-is-a-search-attribute.md

```diff
@@ -29,23 +29,28 @@ A Temporal Cluster has a set of default Search Attributes already available.
+| -------------------------- | -------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| WorkflowType               | Keyword  | The type of Workflow.                                                                                                                                                        |
+| WorkflowId                 | Keyword  | Identifies the Workflow Execution.                                                                                                                                           |
+| ExecutionStatus            | Keyword  | The current state of the Workflow Execution.                                                                                                                                 |
+| StartTime                  | Datetime | The time at which the Workflow Execution started.                                                                                                                            |
+| CloseTime                  | Datetime | The time at which the Workflow Execution completed.                                                                                                                          |
+| ExecutionTime              | Datetime | Same as StartTime for the most cases but different for cron Workflows and retried Workflows. For them it is the time at which the Workflow Execution actually begin running. |
+| RunId                      | Keyword  | Identifies the current Workflow Execution Run.                                                                                                                               |
+| ExecutionDuration          | Int      | The time needed to run the Workflow Execution. Available only for closed Workflows.                                                                                          |
+| HistoryLength              | Int      | The number of events in the history of Workflow Execution. Available only for closed Workflows.                                                                              |
+| StateTransitionCount       | Int      | The number of times that Workflow Execution has persisted its state. Available only for closed Workflows.                                                                    |
+| TaskQueue                  | Keyword  | Task Queue used by Workflow Execution.                                                                                                                                       |
+| TemporalChangeVersion      | Keyword  | If Workflow versioning is enabled, list of change/version pairs will be stored here.                                                                                         |
```

Comment:
That's not the right meaning of TemporalChangeVersion.

Actually as far as I can tell, that search attribute is completely unused and was left over from some cadence thing. I'd suggest removing it from the docs. Maybe @alexshtin knows

Reply:
Sorry, I didn't understand this correctly. It is still used if using the GetVersion api in workflows. I think we should be clear to differentiate this from the newer "build-id based versioning" (both can be used together but are very different things)